app/assets/sass/autoCompleteMultiselects.scss

```diff
@@ -0,0 +1,15 @@
+// Overriding rules from SlimSelect’s stylesheet
+
+.ss-main .ss-multi-selected {
+    border: none;
+}
+
+.ss-main .ss-multi-selected .ss-values .ss-value {
+    font-size: inherit;
+}
```

Comment:
Can we try to add a rule `background-color: #0b0c0c` so that it will be more accessible?

Reply:
Done:

![image](https://user-images.githubusercontent.com/53756884/91553669-b2243300-e925-11ea-8535-a3b7aed00eb9.png)

Reply:
Nice